api/tests/Api/Activities/CreateActivityTest.php

```diff
@@ -471,7 +470,7 @@ public function testCreateActivityValidatesMissingScheduleEntries() {
     public function getExampleWritePayload($attributes = [], $except = []) {
         return $this->getExamplePayload(
             Activity::class,
-            OperationType::COLLECTION,
+            '/activities',
```

Comment:
Couldn't we derive the endpoint from the Entity class? I don't really see why we should have to pass this into `getExamplePayload`. From the perspective of a developer using this method, it is not clear why the payload depends on the endpoint when the Entity class and the operation are already specified in the arguments.

Reply:
Simplified with https://github.com/ecamp/ecamp3/pull/3176/commits/bf6ae18430a9abe940c6ee91448f0748890e0025